src/main/java/uk/gov/nationalarchives/pdi/step/jena/groupmerge/JenaGroupMergeStep.java

```diff
@@ -236,14 +236,16 @@ private void outputGroup(final JenaGroupMergeStepData data) throws KettleStepExc
 
         // Do we want to copy the meta (JenaGroupMergeStepMeta) into this method to get the groupFields
```

Comment:
@enriquedelpino I think we can remove this comment now, once done, we can merge this immediately